spec/integration/post/show_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts show view', type: :system do
+  describe 'Show page' do
+    before(:each) do
+      @user = User.create(name: 'james',
+                          photo: 'https://img.freepik.com/free-psd/save-world-reusable-grocery-bag-mockup_53876-98827.jpg?', bio: 'james')
+      @post = Post.create(text: 'ken', title: 'ken', author: @user)
+      Comment.create(post: @post, author: @user, text: 'Amazing post Tom')
+      Comment.create(post: @post, author: @user, text: 'Awesome post Tom')
+      Comment.create(post: @post, author: @user, text: 'Great post Tom')
+      visit user_post_path(@user.id, @post)
+    end
+
+    it 'Should have a post titkle' do
```

Comment:
- [ ] May you mean title? Please 🙏🏽 write the correct word for professional documentation.